File: Python/Very Easy/calculate_years.py
# https://edabit.com/challenge/y8pj5GjGNarmYQ48d

def calculate_years(human_years):
    cat=0
    dog=0
    if human_years==1:
        cat+=15
        dog+=15
    elif human_years==2:
        cat+=24
        dog+=24
    else:
        cat+=24
        dog+=24
        for number in range(2,human_years):
            cat+=4
            dog+=5
    return [human_years,cat,dog]



print(calculate_years(1), [1,15,15])
print(calculate_years(2), [2,24,24])
print(calculate_years(10), [10,56,64])
print(calculate_years(20), [20,96,114])
print(calculate_years(65), [65,276,339])
print(calculate_years(43), [43,188,229])
print(calculate_years(100), [100,416,514])
#Mubashir
    # elif human_years ==2:
    #     cat+=24
	# 	dog+=24
	# return [human_years,cat,dog]

File: javascript/Very_Easy/calculateYears.js
// https://edabit.com/challenge/HbkxB6egEhRewQDqr

function calculateYears(humanYears) {
    humanYears=(humanYears+1)
    const final =[0,0,0]
    for(let i=1;i<humanYears;i++){
        if(i===1){
            final[0]+=1
            final[1]+=15
            final[2]+=15
        }
        else if(i===2){
            final[0]+=1
            final[1]+=9
            final[2]+=9
        }
        else{
            final[0]+=1
            final[1]+=4
            final[2]+=5
        }
    }
    return final
}



console.log(calculateYears(1), [1,15,15])
console.log(calculateYears(2), [2,24,24])

console.log(calculateYears(10), [10,56,64])
